modules/exploits/windows/misc/hta_server.rb

```diff
@@ -0,0 +1,59 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
```

Comment:
You're right, these aren't necessary any more so I removed them in commit bd11043.